tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain.md

```diff
@@ -0,0 +1,174 @@
+        <span data-ty>2024-09-09 08:32:48 🔖 Pre-sealed block for proposal at 1. Hash now 0xcb3d2f28bc73807dac5cf19fcfb2ac6d7e922756da9d41ca0c9dadbd0e45265b, previously 0xc14630b76907550bef9037dcbfafa2b25c8dc763495f30d9e36ad4b93b673b36.</span>
+        <span data-ty>2024-09-09 08:32:48 🏆 Imported #1 (0x0eef…935d → 0xcb3d…265b)</span>
+    </div>
+
+3. Confirm that your blockchain is producing new blocks by checking if the number after **finalized** is increasing
+    <div id='termynal' data-termynal>
+        <span data-ty>...</span>
+        <span data-ty>2024-09-09 08:32:47 💤 Idle (0 peers), best: #0 (0x0eef…935d), finalized #0 (0x0eef…935d), ⬇ 0 ⬆ 0</span>
+        <span data-ty>...</span>
+        <span data-ty>2024-09-09 08:32:52 💤 Idle (0 peers), best: #1 (0xcb3d…265b), finalized #0 (0x0eef…935d), ⬇ 0 ⬆ 0</span>
+        <span data-ty>...</span>
+        <span data-ty>2024-09-09 08:32:57 💤 Idle (0 peers), best: #2 (0x16d7…083f), finalized #0 (0x0eef…935d), ⬇ 0 ⬆ 0</span>
+        <span data-ty>...</span>
+        <span data-ty>2024-09-09 08:33:02 💤 Idle (0 peers), best: #3 (0xe6a4…2cc4), finalized #1 (0xcb3d…265b), ⬇ 0 ⬆ 0</span>
+        <span data-ty>...</span>
+    </div>
+
+    !!!note
+        The details of the log output will be explored in a later tutorial. For now, it's sufficient to know that your node is running and producing blocks.
+
+## Interact with the Node
+
+When running the template node, it's accessible by default at:
+
+```bash
+ws://localhost:9944
+```
+To interact with your node using the [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} interface, follow these steps:
+
+1. Open [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} in your web browser and click the network icon in the top left corner
+    ![](/images/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/build-a-local-blockchain-1.webp)
+
+2. Connect to your local node:
+    1. Scroll to the bottom and select **Development**
+    2. Choose **Custom**
+    3. Enter `ws://localhost:9944` in the input field
+    4. Click the **Switch** button
+    ![](/images/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/build-a-local-blockchain-2.webp)
+
+3. Verify Connection:
+    - Once connected, you should see **solochain-template-runtime** in the top left corner
+    - The interface will display information about your local blockchain
+    ![](/images/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/build-a-local-blockchain-3.webp)
+
+You are now connected to your local node and can interact with it through the Polkadot.js Apps interface. This interface allows you to explore blocks, make transactions, and interact with your blockchain's features.
+
+## Transfer Funds
```

Comment:
I'd remove this entire section since it will be covered by the polkadot js transfer guide

Reply:
Not sure about this one. I think it is important for the one who is following the tutorial to understand how to interact with it

Reply:
If there is already going to be a transfer guide, or even other PJS guides, you can link it in a manner like: 

"Follow these tutorials to interact with your chain", and you can link specific ones, etc.

Reply:
Makes sense. These guides will stay in the Polkadot Wiki? Nico started migrating the Transfer guides but I think we stopped because they were 'end user' tutorials

Reply:
Yeah, my suggestion is to link back to the existing page on the Wiki for now and add a comment on the line above or below the link like `<!-- TODO: Polkadot.js Apps guide -->` so we can easily find where we need to make updates once we need to make them